Anlab.Mvc/Controllers/OrderController.cs

```diff
@@ -235,9 +235,32 @@ public OrderController(ApplicationDbContext context, IOrderService orderService,
 
             await _context.SaveChangesAsync();
 
-            Message = "Order confirmed";
-            return RedirectToAction("Index");
+            return RedirectToAction("Confirmed", new { id = id });
+
+        }
+
+        public async Task<IActionResult> Confirmed(int id)
+        {
+            var order = await _context.Orders.Include(i => i.Creator).SingleOrDefaultAsync(o => o.Id == id);
+
+            if (order == null)
+            {
+                return NotFound();
+            }
+
+            if (order.CreatorId != CurrentUserId)
+            {
+                ErrorMessage = "You don't have access to this order.";
+                return NotFound();
+            }
+
+            var model = new OrderReviewModel();
+            model.Order = order;
+            model.OrderDetails = order.GetOrderDetails();
+            model.TestItems = _context.TestItems

```

Comment:
@laholstege do we really need to put the testItems here?  aren't they stored in the order details already?  If we don't need it we can save an expensive deserialization step